gui/util/range_slider.py

```diff
@@ -172,30 +177,43 @@ def updateScaleValues(self):
 class QHRangeSlider(QRangeSlider):
     """
     Horizontal Range Slider, extended from QRangeSlider
-    """
 
+    Parameters
+    ----------
+    slider_range : 3-tuple of int
+        Min, max, and step of the slider.
+    values : 2-tuple of int
+        Start and end of the slider range.
+    parent : PyQt5.QtWidgets.QWidget
+        Parent widget.
+    """
     def __init__(self, slider_range=None, values=None, parent=None):
-        """
-        Constructor of the QHRangeSlider class
-        :param slider_range: Should be a list of three elements such: [min, max, step]. Default value is None and will be inherited from QRangeSlider.
-        :param values: Should be a list of two elements such: [beginning, end]. Default value is None and will be inherited from QRangeSlider.
-        :param parent: Shall be passed only if explicit call to parent constructor is needed.
-        """
         QRangeSlider.__init__(self, slider_range, values, parent)
         if not parent:
             self.setGeometry(200, 200, 200, 20)
 
     def getPos(self, event):
-        """
-        :param event: catches from qt context
-        :return: relative horizontal position of event
+        """Get event position.
+
+        Parameters
+        ----------
+        event : ?
```

Comment:
qt eventloop constantly working. This methods is invoked whenever it is needed for the event of the moment. So event: Event from Qt context. That's true.

Reply:
Yeah but is there an exact `QtEvent` class or something like that?

Reply:
There is base class `QEvent`: http://doc.qt.io/qt-5/qevent.html